db/migrate/20150819105725_add_lookup_value_match_to_host_and_hostgroup.rb

```diff
@@ -0,0 +1,23 @@
+class AddLookupValueMatchToHostAndHostgroup < ActiveRecord::Migration
+  def up
+    add_column :hosts, :lookup_value_matcher, :string
+    add_column :hostgroups, :lookup_value_matcher, :string
+
+    Host::Managed.where(:type => 'Host::Managed').find_in_batches(:batch_size => 100) do |group|
```

Comment:
`find_each` should suffice if you want to use batches (find_each uses 1000), did you find this migration slow?